packages/opencensus-core/test/test-stats-aggregation.ts

```diff
@@ -0,0 +1,85 @@
+/**
+ * Copyright 2018, OpenCensus Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import * as assert from 'assert';
+
+import {Histogram, MeasureType, StatsAggregation, StatsBucketBoundaries, StatsManager, StatsMeasure, StatsMeasurement, StatsView, Tags} from '../src/index';
+
+const manager = new StatsManager();
```

Comment:
Every test object should be initialized in a `describe` clause rather than at the top level.

Reply:
Done.